src/server.ts

```diff
@@ -22,6 +40,18 @@ router.get('/test', (req: any, res: any) => res.send("Hello World!"));
 
 router.get("/rovers", (req, res) => getRovers(res));
 
+router.get("/rovers/:rovername/photos/:cameratype", (req, res: any) => {
+    let rovername: rover = rover[req.params["rovername"] as keyof typeof rover] // makes sure rover name is in list of rovers
+    let cameratype: camera = camera[req.params["cameratype"] as keyof typeof camera] // makes sure camera type is in list of camera types

```

Comment:
If you're not reassigning the values of a variable then its best practice to make them `const` instead. This means that if you accidentally do a function that tries to change your variable, TS will catch it and tell you something like `you can't reassign the value of a const`